e2e/tests.go

```diff
@@ -95,26 +96,6 @@ func main() {
 		logrus.Info("Installation succeeded")
 	}
 
-	// reconfigure CR to enable TLS support
-	logrus.Info("Patching CR with TLS enabled. This should cause a new Che deployment")
```

Comment:
I guess the code was removed because you covered that with bash script, correct?

Reply:
Yes. But in the future probably need to improve tls tests coverage.